services/dashboard/app/routes/_index.tsx

```diff
@@ -1,17 +1,25 @@
 import { type MetaFunction } from '@remix-run/node'
+import {Link} from "@remix-run/react"
 
 export const meta: MetaFunction = () => [{ title: 'PlanEs' }]
 
 export default function Index() {
+	
 	return (
 		<main className="relative flex min-h-screen items-center justify-center">
 			<div className="relative">
+			<div className='display: flex items-center justify-center'>
```

Comment:
Do you need the `display: ` className?
```suggestion
			<div className='flex items-center justify-center'>
```